docs: document available models (#13)
gravitee-io/gravitee-resource-ai-model-text-classification: README.adoc

* link:https://huggingface.co/gravitee-io/Llama-Prompt-Guard-2-86M-onnx[gravitee-io/Llama-Prompt-Guard-2-86M-onnx]

``Llama 4 is licensed under the Llama 4 Community License, Copyright © Meta Platforms, Inc. All Rights Reserved.``

== Available models

The AI model text classification resource provides the following models:

- minuva/MiniLMv2-toxic-jigsaw-onnx(https://huggingface.co/minuva/MiniLMv2-toxic-jigsaw-onnx)
- gravitee-io/detoxify-onnx(https://huggingface.co/gravitee-io/detoxify-onnx)
- gravitee-io/distilbert-multilingual-toxicity-classifier(https://huggingface.co/gravitee-io/distilbert-multilingual-toxicity-classifier)
- gravitee-io/Llama-Prompt-Guard-2-22M-onnx(https://huggingface.co/gravitee-io/Llama-Prompt-Guard-2-22M-onnx)
- gravitee-io/Llama-Prompt-Guard-2-86M-onnx(https://huggingface.co/gravitee-io/Llama-Prompt-Guard-2-86M-onnx)

NOTE: The models are not included in the resource package. The resource will automatically download the models from Hugging Face when they are not available locally. Models files are store in `$GRAVITEE_HOME/models` directory, please ensure the user running the Gateway is allowed to write in that directory.

> WARNING: Models are loaded in memory, so you may need to increase the memory (Java Heap size and Kubernetes resource limits) of the Gateway to avoid any OutOfMemory error. The increase will depend on the model size and the number of APIs using this kind of resource.

Reviewer: ```suggestion
WARNING: Models are loaded in memory. You may need to increase the memory (Java Heap size and Kubernetes resource limits) of the Gateway to avoid any OutOfMemory error. The increase depends on the model size and the number of APIs that use the resource.
```
